src/libs/Middleware/SaveResponseInOnyx.ts

```diff
@@ -9,6 +9,7 @@ const requestsToIgnoreLastUpdateID: string[] = [
     WRITE_COMMANDS.OPEN_APP,
     SIDE_EFFECT_REQUEST_COMMANDS.RECONNECT_APP,
     WRITE_COMMANDS.CLOSE_ACCOUNT,
+    WRITE_COMMANDS.DELETE_MONEY_REQUEST,
```

Comment:
@jasperhuangg @aimane-chnaif @huult hey guys I noticed this change while working on another issue. Similar to another instance [here](https://github.com/Expensify/App/pull/53541#discussion_r1893074416), this is probably the wrong way to fix the bug we were addressing here. This disables reliable updates for `DeleteMoneyRequest` which can cause reliability issues. Can we revert this and try fixing the root problem another way?

Reply:
@arosiclair We need a backend update. If lastUpdateIDFromClient is less than the previousUpdateID received, the client needs updating.

Reply:
I'm not sure what change is needed specifically, but `lastUpdateIDFromClient` and `previousUpdateID` are not the root problem. Instead we probably need to fix the onyx updates themselves or maybe the way we're rendering them in React.